docs/Installing.asciidoc

```diff
@@ -336,6 +338,13 @@ required. This will direct all HTTP traffic to openQA.
 cp /etc/apache2/vhosts.d/openqa.conf.template /etc/apache2/vhosts.d/openqa.conf
 --------------------------------------------------------------------------------
 
+=== NGINX proxy
```

Comment:
https://nginx.org/en/ writes it with lowercase so I would use `nginx proxy` here as well